### PR TITLE
Update to configure JwkProvider for use with RS256 tokens

### PR DESCRIPTION
**Pull request by jimmyjames**

### CODE REVIEW

**Comment by jimmyjames** [comment]
QS Article PR here: https://github.com/auth0/docs/pull/8440

File: 01-Login/README.md [hunk 1]
```diff
@@ -28,15 +28,18 @@ com.auth0.clientId: {YOUR_AUTH0_CLIENT_ID}
 com.auth0.clientSecret: {YOUR_AUTH0_CLIENT_SECRET}
 ```
 
-It will request by default a `code` Response Type and later execute a Code Exchange. You can modify this behavior by changing the Response Type in the `AuthController` class to `token` or `id_token` to use Implicit Grant. i.e.:
+By default, `mvc-auth-commons` uses the Authorization Code flow and assumes tokens are signed with the HS256 signing algorithm.
+
+If using RS256 (recommended, and the default for new applications), you need to configure the `AuthenticationController` with a `JwkProvider` to fetch the public signing key used to verify the ID token:
```
Comment (joshcanhelp) [inline 01-Login/README.md]: Do we need to say all this? Probably good to have them check the Auth0 Application but maybe skip even talking about HS at all?

**Review by joshcanhelp — APPROVED** [review]
Approving for now but comment here is in the context of the docs PR. Ping me if you need a re-review.

File: 01-Login/README.md (hunk 1 above)
Comment (jimmyjames) [reply 01-Login/README.md]: That's a good point. I've pushed a new commit to simplify the README.

**Pull request merged**